### ISSUE TITLE
Vortex puts large files in %APPDATA%.

### ISSUE THREAD

**Issue by MicahZoltu** [Nexus-Mods/Vortex#11736]
**Describe the bug**
Vortex is putting large files into `%APPDATA%`, which is against the OS guidelines and recommendations and causes problems in a number of environments (anything where data roams over the network).

**To Reproduce**
1. Join computer to a domain that uses roaming profiles
2. download some large mods
3. Log out
4. go to a different computer in the same domain.
5. Log in
6. observe bug

**Expected behavior**
Vortex stores large files in `%LOCALAPPDATA%` (per user) or `%PROGRAMDATA%` (all users).

**Platform (please complete the following information):**
 - OS: Windows 7/8/9/10/11
 - Vortex Version: 1.5.10

**Additional context**
This is a recreate of #5589 which was closed but the reason for closure is incorrect.  I'm opening a new issue to make sure this gets seen by the Vortex dev team.

For context, the closure comment in #5589 was:
> This is the intended behavior. AppData/Local according to MS is intended only for data that is specific to the current PC, usually stuff like caches or activity history.
> The Vortex data isn't system specific, it's completely possible you may want to synchronize your mods between systems and play the same setup on different machines. If not you can change the storage directories in settings or you can configure your roaming profile to exclude the vortex dir.

TL;DR: Microsoft has a lot of documentation on this topic and `%APPDATA%` is definitely not the correct place to store 100s of GB of data.

`%LOCALAPPDATA%` is anything that should not be copied over the network on login/logout but is user specific.  `%APPDATA%` is copied/synced over the network on every login/logout and regularly backed up to cloud backup providers.  `%LOCALAPPDATA%` is meant for user-specific files that should not roam over the network for any reason, because they are machine specific or because they are too large to reasonably roam (anything over ~1MB is too large to reasonably roam, some rare exceptions apply), or because they are caches and other files that can be regenerated (in this case, these are mostly files that can be regenerated, see below).

While I agree that syncing mods between machines could be valuable for some users, this should be handled by Vortex not by the OS.  The OS's roaming system isn't designed to handle 100GB of data synced between every machine the user logs into on every login/logout.  Instead, a manifest could be stored in `%APPDATA%` which is synced over the network and then on launch Vortex could read the manifest and then ensure the mods (stored in `%LOCALAPPDATA%` match what the manifest says, potentially re-downloading some of them.  This would prevent the user from syncing 100GB of data to every machine they connect to despite the fact that only one of them has Vortex installed.

It is worth noting that in Vortex preferences I was only able to move the downloads folder to another location.  While this did get rid of *most* of the content in %APPDATA%, it still left several hundred megabytes of other stuff behind which is still far too much to roam.  At the least, Vortex should allow you to move everything out of `%APPDATA%`.

**Comment by TanninOne** [comment]
I've already explained the reason why it works the way it does in the referenced ticket.
